src/friktion/index.ts

```diff
@@ -4,6 +4,14 @@ export * from "./layouts";
 import { PublicKey } from "@solana/web3.js";
 import { IDepositorInfo, IVaultInfo, IWithdrawerInfo } from "../types";
 import BN from "bn.js";
+
+export enum VaultType {
+  LongBasis = "LongBasis",
```

Comment:
Is the string type enum necessary? Can just use integer? Ex: 

```typescript
enum Foo {
  Bar,
  Baz,
}
```

Reply:
If it's using a integer enum, when `console.log(vault.getTypes())`, the log will be `0 1 2`,
it's not easy to read and debug using a number type enum

Reply:
Noted